lib/injekt.rb

```diff
@@ -0,0 +1,29 @@
+class Array
+
+  def injekt(*args, &block)
+
+    len = args.length
+    message = "wrong number of arguments (#{len} for 0..2)"
+    raise ArgumentError.new(message) if len >= 3
```

Comment:
Style/RaiseArgs: Provide an exception class and message as arguments to `raise`.<br>Style/SignalException: Use `fail` instead of `raise` to signal exceptions.